Automattic/pocket-casts-ios
# [1366] Add internal Privacy Manifest

Fixes #1366

[As Apple last updates](https://href.li/?https://developer.apple.com/news/?id=3d8a9yyh) we added the Privacy Manifests to these folders:
- PodcastsIntents
- WidgetExtension
- Pocket Casts Watch App Extension
- podcasts

## To test

- App should build as normal and it should not generate new warnings
- CI should be 🟢 
- Double check the new Privacy Manifest and the API type + reasons:

#### PodcastsIntents
- Type: NSPrivacyAccessedAPICategoryUserDefaults
- Reasons: 1C8F.1

#### WidgetExtension
- Type: NSPrivacyAccessedAPICategoryUserDefaults
- Reasons: 1C8F.1

#### Pocket Casts Watch App Extension
- Type: NSPrivacyAccessedAPICategoryUserDefaults
- Reasons: 1C8F.1
- Type: NSPrivacyAccessedAPICategoryFileTimestamp
- Reasons: DDA9.1

#### podcasts
- Type: NSPrivacyAccessedAPICategoryUserDefaults
- Reasons: 1C8F.1
- Type: NSPrivacyAccessedAPICategoryFileTimestamp
- Reasons: DDA9.1
- Type: NSPrivacyAccessedAPICategoryDiskSpace
- Reasons: E174.1

For the reasons code, please refer to [this Apple doc](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api)

## Checklist
